modules/subnet_router/variables.tf

```diff
@@ -12,6 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+variable "name" {
```

Comment:
@cablespaghetti You need to move these changes into the root `variables.tf` ([this guy][1]) as well

[1]: https://github.com/hardfinhq/terraform-aws-tailscale-subnet-router/blob/main/variables.tf